src/main.js

```diff
@@ -494,31 +494,36 @@ var lastChar;
 
 function logWithPrefix(prefix, prefixColor, text, color) {
 
-     if (config.raw) {
+    if (config.raw) {
         process.stdout.write(text);
         return;
     }
 
     text = text.replace(/\u2026/g,'...'); // Ellipsis
+    text = text.replace(/\r/g, '');
```

Comment:
Would it still work for you if `\n\r` was replaced directly, instead of removing all `\r`?